src/familiar/lib.ts

```diff
@@ -79,3 +82,12 @@ export function canOpenRedPresent(): boolean {
     myInebriety() <= inebrietyLimit()
   );
 }
+
+export function turnsAvailable(): number {
+  const baseTurns = estimatedTurns();
+  const digitizes = digitizedMonstersRemaining();
+  const mapTurns = globalOptions.ascending
+    ? clamp(availableAmount($item`Map to Safety Shelter Grimace Prime`), 0, 60)
```

Comment:
so i think clamping mapTurns to 60 is totally fine, i know garbo clamps actually generating the maps to 60 [here](https://github.com/Loathing-Associates-Scripting-Society/garbage-collector/blob/dc4a38ceeb96166bd6122c230b06c27bce341098/src/fights.ts#L2152) but do we maybe want to tie this & the max number of maps garbo grabs to one number so we don't have to change the code in multiple places in the event we someday get a better nightcap? maybe like "export const expectedOverdrunkTurns = 60;" above this and clamp both this & the value in fights.ts to that constant?

Reply:
Yeah I'll name this constant instead of using a magic number